questions.md

```diff
@@ -0,0 +1,7 @@
+// Questions: 
+- in yml file:
+  Action:
+    - sqs:* 
```

Comment:
`*` allows all actions. You can be specific and say write. The best practice is the principle of least privilege. But this is fine for now.